index.yaml

```diff
@@ -31,25 +31,28 @@ pages:
     title: Releases
     file: releases.md
     description: Past, current, and future release information.
-  - name: contributing
-    title: Contributing
-    file: contributing.md
-    description: How to contribute to Gazebo.
+  - name: development 
+    title: Development
+    file: development.md
+    description: Gazebo development information
+    children:
+      - name: ci
+        title: Continuous Integration
+        file: ci.md
+        description: Continuous Integration overview.
+      - name: contributing
+        title: Contributing
+        file: contributing.md
+        description: How to contribute to Gazebo.
+      - name: release
+        title: Release process
+        file: release.md
+        description: Release process overview.
   - name: architecture
-    title: Architecture
+    title: Sim Architecture
     file: architecture.md
     description: Gazebo Sim's architecture overview.
     unlisted: true
```

Comment:
Do we want to get rid of the `unlisted`?

Reply:
removed.